docs/proposals/202003_thanos_rules_federation.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Global / Federated Rules API
+type: proposal
+menu: proposals
+status: accepted
+owner: s-urbaniak,bwplotka
+---
+
+### Related Tickets
+
+https://github.com/thanos-io/thanos/pull/2200
```

Comment:
I would argue that this is not related, it is your actual implementation since you started doing this backwards. This should probably have https://github.com/thanos-io/thanos/issues/1375.

Reply:
That's true!

Reply:
Agreed I'll swap the referenced #2200 PR with https://github.com/thanos-io/thanos/issues/1375, although the latter is about federating targes but this proposal is about federating rules.